Justfile

```
@@ -183,3 +181,15 @@ conformance-status:
 
 conformance-logs:
   sonobuoy logs
+
+dev-conformance *ARGS:
+  devspace dev --profile test-conformance --namespace vcluster {{ARGS}}
+
+create-conformance k8s_version="1.31.1":
+  minikube start --kubernetes-version {{k8s_version}} --nodes=2
+  minikube addons enable metrics-server
+
+delete-conformance:
+  minikube delete || true
```

Just can [ignore errors](https://github.com/casey/just#ignoring-errors) by prefixing the line with a dash

```suggestion
  -minikube delete
```